src/apps/replaceRootSources/replaceRootSources.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import json
+
+def replace_root_sources(lock, newSource):
+
+    packages = lock['_generic']['packages']
+    sources = lock['sources']
+
+    for name, version in packages.items():
+
+        original = sources[name][version]
+
+        fixed = newSource
+
+        if 'dir' in original:
+            fixed['dir'] = original['dir']
+        elif 'relPath' in original:
+            fixed['dir'] = original['relPath']
```

Comment:
@yusdacra I saw that the `relPath` thing comes from your original implementation. But what is the purpose? According to `src/specifications/dream-lock-schema.json`, relPath cannot occur in a source.
Also, it would be nice if you could review the rest of this function. Thanks

Reply:
relPath was used before in `path` sources iirc, but it seems to have been removed so i don't think we need to have a check for that here

Reply:
Removed the `relPath` check